Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -3252,6 +3254,7 @@ static NSStringCompareOptions compareOptionsForFindOptions(WebCore::FindOptions
         m_lastFindString = target;
         setCurrentSelection(nullptr);
         m_findMatchRects.clear();
+        webFoundTextRangePDFDataSelectionMap().clear();
```

Comment:
This will clear the map when a new search is performed, but if you have one in progress and then close the find bar, we'd still be keeping around the cache contents right? Is that okay?

Reply:
We wouldn't be, because closing the find bar calls into `hideFindUI()` too, which calls `UnifiedPDFPlugin::findString("", ...)`

Reply:
So this should be fine

Reply:
Nice!